### DISCUSSION TITLE
dose vercel support upload files directlry from client to ftp ?

### DISCUSSION THREAD

**Discussion by m0x61h0x64i** [vercel/vercel#8758] [Help] [answered]
hello, i know that vercel has upload payload limit of 4.5 MB, and my question here is that dose vercel support ftp ?
i want to uplaod client files to ftp. is that possible ?
what are the available ways that i can upload files directly from client to a server or cloud server or cloud storage etc... and bypass the max 4.5 MB payload limit ?

**Comment by amyegan** [comment, marked as answer]
Hi @m0x31h0x64i! You should take a look at the [File Storage & Uploads](https://vercel.com/docs/concepts/solutions/file-storage#) doc. You can also learn more about how to work around the payload limit in this guide: [How do I bypass the 4.5MB body size limit of Vercel Serverless Functions?](https://vercel.com/guides/how-to-bypass-vercel-body-size-limit-serverless-functions)
